testsupport/wait/awaitility.go

```diff
@@ -539,6 +539,15 @@ func UntilToolchainClusterHasCondition(expected toolchainv1alpha1.ConditionType)
 	}
 }
 
+// UntilToolchainClusterHasCondition checks if ToolchainCluster has the given condition and False Status
+func UntilToolchainClusterHasConditionAndFlaseStatus(expected toolchainv1alpha1.ConditionType) ToolchainClusterWaitCriterion {
```

Comment:
hellooo! typo minor suggestion:
```suggestion
func UntilToolchainClusterHasConditionAndFalseStatus(expected toolchainv1alpha1.ConditionType) ToolchainClusterWaitCriterion {

```

Reply:
thanks updated it